### ISSUE TITLE
Failed to import command at 'vollseg_napari._dock_widget:plugin_wrapper_vollseg': No module named 'segment_anything'

### ISSUE THREAD

**Issue by dsethz** [kapoorlab/vollseg-napari#9]
Hey,

I tried to install `vollseg-napari` from command line in a fresh environment that has napari already installed. The installation proceeds without error, but when I want to open the plugin in napari I get the following error:

```
Failed to import command at 'vollseg_napari._dock_widget:plugin_wrapper_vollseg': No module named 'segment_anything'
```

Tested environments:
Windows 10
python 3.10.12
napari 0.4.18
vollseg-napari 2.4.7/2.4.3/2.3.8/2.2.8

Thank you

**Comment by kapoorlab** [comment]
Yes, segment-anything is a requirement coming from vollseg package, I suspect that your vollseg package is outdated so you can manually install the latest version by `pip install vollseg-napari --ignore-installed` ,or you can just install segment-anything using pip: `pip install segment-anything`. If that resolves the issue please close this issue.

**Comment by dsethz** [comment]
Thank you for the fast response.

I was using the latest version of vollseg-napari released on pypi (2.4.7). Beyond `segment-anything`, `torch` and `torchvision` needed to be installed manually. That resolved the issue.
